db.json

```diff
@@ -119,5 +119,27 @@
         "about":"-",
         "homepage":"https://github.com/PyThaiNLP/pythainlp-corpus",
         "authors":"Wannaphong Phatthiyaphaibun"
+    },
+        "male_names_th":{
+        "name":"male_names_th",
+        "file_name":"male_names_th.txt",
+        "version":"0.1",
+        "download":"https://raw.githubusercontent.com/korkeatw/thai-names-corpus/master/male_names_th.txt",
```

Comment:
it would be great if we could incorporate version into the source path.

One way to do this is to use `git tag`. Then, one could a url like the one below:
```
download: "https://github.com/---corpus/tag/0.1/male_names_th.txt
```

A nicer approach would be to use a url template with `someurl/{version}/male_names_th.txt` and modify `pythainlp-corpus` to handle this string format.

Reply:
It is good to think about version organization, seems more automate, I like this idea but we cannot do string template here. The following are my concerns:

- The `db.json` is just JSON file so we cannot program anymore. If we use string template here, it requires code changes in PyThaiNLP repo
- Even we do code changes to support string template in PyThaiNLP repo, some courpuses won't be loaded because other corpuses don't be tagged the version (I saw some of them still use `master` branch)

Perhaps, we merge this first and revise it later.